### ISSUE TITLE
Local folders

### ISSUE THREAD

**Issue by popliviustefan** [nextcloud/mail#4602]
### Feature Request

Add possibility to create local folders in the Email app.

#### Summary

This would be a useful feature which would make the Nextcloud Email a very robust alternative to a desktop email client like Mozilla Thunderbird.
You already have the possibility to have multiple accounts and most of the functionality in a desktop client, but the Local Folders it is kind of needed to be able to completely drop any other email client for general, non-specific usage.

The implementation should be similar to that in most popular desktop clients (to allow to create a tree of local folders, move/copy emails from/to the other accounts registered in the Mail app and search).

The storage for those local folders should be part of the storage quota of the user to avoid abuses and respect the general philosophy of nextcloud.

**Comment by ChristophWurst** [comment]
I agree that this would be a cool feature, but it comes with a lot of complexity. Right now this is too much of a stretch and I can't promise that we ever work on this.

**Comment by sunjam** [comment]
Perhaps time to add the "nice to have" tag

**Comment by Extarys** [comment]
Darn, I needed that because we have way too many emails on our mail service provider and wanted an easy way to still access them without having them take space on a remote server.

:crossed_fingers: Hope this feature eventually get some love
